contrib/scripts/bugtool-multinode-gather.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Usage
+# ./bugtool-multinode-gather.sh <OUTPUT_DIR> <NAMESPACE> <LABEL> <OUTPUT_ARCHIVE>
+#
+# Running without arguments is the same as
+# 
+# ./bugtool-multinode-gather.sh `mktemp -d` kube-system k8s-app=cilium "`mktemp`.tar.gz"
+
+OUTPUT_DIR=${1:-"`mktemp -d`"}
+NAMESPACE=${2:-"kube-system"}
+LABEL=${3:-"k8s-app=cilium"}
+OUTPUT_ARCHIVE=${4:-"`mktemp`.tar.gz"}
+
+mkdir -pv ${OUTPUT_DIR}
+function cleanup {
+  if [ "${OUTPUT_DIR}" == "/" ]; then
+    echo "Aborting cleanup OUTPUT_DIR=${OUTPUT_DIR}"
+    exit 0
+  fi
+  rm -rf ${OUTPUT_DIR}
+}
+trap cleanup EXIT
+
+for CILIUM in $(kubectl -n ${NAMESPACE} get pods --selector=${LABEL} --output=jsonpath={.items..metadata.name}); do
+  BUGTOOL_CMD="cilium-bugtool --archive=false"
+  echo "=============== ${CILIUM} ${BUGTOOL_CMD} ==============="
+  ARCHIVE=`kubectl exec -n ${NAMESPACE} ${CILIUM} -- ${BUGTOOL_CMD} | grep DIRECTORY | awk '{ print $3}'`
```

Comment:
On the assumption that mktemp and friends are random enough, would it be more resilient to pass in the output directory to cilium-bugtool instead of trying to read it back out? This make the log output less of a public API the  cli tool has to maintain (and uses the set output dir codepath in tests and things)